schemas/appeal-s78.schema.json

```diff
@@ -0,0 +1,1078 @@
+    },
+    "appellantPreferHearingDetails": {
+      "type": ["string", "null"],
+      "description": "The hearing details preference indicated by the appellant",
+      "$comment": "Only available in S78 cases"
+    },
+    "appellantPreferInquiryDetails": {
+      "type": ["string", "null"],
+      "description": "The inquiry details preference indicated by the appellant",
+      "$comment": "Only available in S78 cases"
+    },
+    "appellantPreferInquiryDuration": {
+      "type": ["number", "null"],
+      "description": "The duration of enquiry indicated by the appellant",
+      "$comment": "Only available in S78 cases"
+    },
+    "updateDevelopmentDescriptionUploaded": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that an update development description document has been submitted",
+      "$comment": "Only available in S78 cases"
+    },
+    "ownershipCertificateSubmitted": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that an ownership certificate has been submitted",
+      "$comment": "Only available in S78 cases"
+    },
+    "designAccessStatementProvided": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that a design and access statement has been provided",
+      "$comment": "Only available in S78 cases"
+    },
+    "newPlansDrawingsProvided": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that new plans or drawings have been provided",
+      "$comment": "Only available in S78 cases"
+    },
+    "planningObligation": {
+      "type": ["boolean", "null"],
+      "description": "Indicates the existence of a planning obligation",
+      "$comment": "Only available in S78 cases"
+    },
+    "statusPlanningObligation": {
+      "type": ["string", "null"],
+      "description": "The planning obligation information, if available",
+      "$comment": "Only available in S78 cases"
+    },
+    "appellantFinalCommentDetails": {
+      "type": ["string", "null"],
+      "description": "The final comments details indicated by the appellant",
+      "$comment": "Only available in S78 cases"
+    },
+    "appellantWitnesses": {
+      "type": ["boolean", "null"],
+      "description": "Indicates if there are any witnesses",
+      "$comment": "Only available in S78 cases"
+    },
+    "affectsScheduledMonument": {
+      "type": ["boolean", "null"],
+      "description": "Indicates that a scheduled monument is affected",
+      "$comment": "Only available in S78 cases"
+    },
+    "hasProtectedSpecies": {
+      "type": ["boolean", "null"],
+      "description": "Indicates the existence of protected species",
+      "$comment": "Only available in S78 cases"
+    },
+    "isAonbNationalLandscape": {
+      "type": ["boolean", "null"],
+      "description": "Indicates an area of outstanding beauty (National Landscape)",
+      "$comment": "Only available in S78 cases"
+    },
+    "designatedSitesNames": {
+      "type": ["string", "null"],
```

Comment:
this has an enum of options + an other field for an anything else
perhaps add as examples
["SSSI", "cSAC", "SAC", "pSPA", "SPA", "user entered value"]